include/benchmark/benchmark_api.h

```diff
@@ -414,16 +416,18 @@ class State {
   // Range arguments for this run. CHECKs if the argument has been set.
   BENCHMARK_ALWAYS_INLINE
   int range_x() const {
```

Comment:
we do not. a comment will do for now, or you could just remove them. anyone building this from head should be open to breaking changes, and we'll make sure to add this to any release notes for future releases.